src/nodes/toJS.ts

```diff
@@ -33,13 +39,24 @@ export function toJS(value: any, arg: string | null, ctx?: ToJSContext): any {
   if (Array.isArray(value)) return value.map((v, i) => toJS(v, String(i), ctx))
   if (value && typeof value.toJSON === 'function') {
     // eslint-disable-next-line @typescript-eslint/no-unsafe-call
-    if (!ctx || !hasAnchor(value)) return value.toJSON(arg, ctx)
+    if (!ctx) return value.toJSON(arg, ctx)
+
+    if (!hasAnchor(value)) {
+      if (ctx.anchorAndAliasNodes && isAlias(value))
+        ctx.anchorAndAliasNodes.push(value)
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+      return value.toJSON(arg, ctx)
+    }
```

Comment:
Could this also be done within the Alias `resolve()`?

Reply:
Yep, moved